src/style.css

```diff
@@ -1 +1,107 @@
-/* add styles here */
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.search__title--big {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+.search__title--big:hover {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  box-shadow: 4px 0 4px 4px #00000040;
+  border: 1px solid #e1e7ed;
+}
+
+.search__link--big {
+  width: 19px;
+  height: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search__big {
+  text-align: left;
+  border: none;
+  font-size: 16px;
+  line-height: 16px;
+  margin-left: 35px;
+  width: 158px;
+  height: 22px;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search__big:focus {
+  font-size: 16px;
+  outline-color: transparent;
+  outline-style: solid;
+  box-shadow: 0 0 0 2px #5a01a7;
+  transition: 0.7s;
+  border-radius: 5px;
+}
+
+.search__title--small {
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+.search__title--small:hover {
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  box-shadow: 4px 0 4px 4px #00000040;
+  border: 1px solid #e1e7ed;
```

Comment:
Similar to the previous comment, the styles for `.search__title--small:hover` are mostly identical to `.search__title--small`. Remove redundant styles to keep the CSS clean and efficient.